plugins/systemd/systemd.plugin.zsh

```diff
@@ -20,3 +20,14 @@ alias sc-mask-now="sc-mask --now"
 alias scu-enable-now="scu-enable --now"
 alias scu-disable-now="scu-disable --now"
 alias scu-mask-now="scu-mask --now"
+
+function sc-prompt-status {
+  for UNIT in $@
+  do
```

Comment:
Put this in the line above:
```zsh
for unit in $@; do
```
Also, variables that change should be lowercase (`$unit`).